core/src/main/java/io/grpc/internal/ManagedChannelImpl.java

```diff
@@ -764,10 +769,13 @@ public void onUpdate(List<ResolvedServerInfoGroup> servers, Attributes config) {
         onError(Status.UNAVAILABLE.withDescription("NameResolver returned an empty list"));
         return;
       }
+      log.log(Level.FINE, "[{0}] resolved address: {1}, config={2}",
+          new Object[] {getLogId(), servers, config});
 
       try {
         balancer.handleResolvedAddresses(servers, config);
       } catch (Throwable e) {
+        log.log(Level.WARNING, "[" + getLogId() + "] Caught exception from LoadBalancer", e);
```

Comment:
If this is unexpected, could you put that in the message?  This [issue](https://github.com/grpc/grpc-java/issues/2604) from yesterday came from someone not realizing they should have been catching the exception, and reading the log message.